examples/hello_world/config/BUILD

```diff
@@ -45,5 +45,5 @@ serialized_config(
         "app": "//:target/wasm32-unknown-unknown/release/hello_world.wasm",
         "translator": "//:target/wasm32-unknown-unknown/release/translator.wasm",
     },
-    textproto = ":config_translator.textproto",
+    textproto = ":config.textproto",
```

Comment:
Why the change here?

Reply:
I deleted the other config, since now they are identical. I can keep both if you prefer though, let me know.